langgraph/pregel/read.py

```diff
@@ -121,11 +121,17 @@ def get_writers(self) -> list[Runnable]:
             and isinstance(writers[-2], ChannelWrite)
         ):
             # we can combine writes if they are consecutive
-            writers[-2].writes += writers[-1].writes
+            # careful to not modify the original writers list or ChannelWrite
```

Comment:
this was a bad bug, which was harmless because of a different bug fixed in write.py